pkg/helm/charts/tests/securedclusterservices/testdata/helmtest/psp-autosense.test.yaml

```diff
@@ -0,0 +1,20 @@
+tests:
+  - name: "Detect support for PodSecurityPolicies"
+    values:
+      meta:
+        apiServer:
+          overrideAPIResources:
```

Comment:
Interesting, never saw this, would like to learn a bit about it. 
How does this work? Can you share a documentation link?

Reply:
Malte came up with this. It's an abstraction layer on top of Helm's lookup functionality which is quite convenient to use and valuable in particular for testing purposes, allows mocking (un)availability of APIs.

I don't have a pointer to documentation for this unfortunately.